login.html

```diff
@@ -26,29 +52,41 @@
 
       <main>
         <div class="main__wrap">
-          <form class="login-form">
+          <form class="login-form js-form">
             <label for="email" class="login-form__label">이메일</label>
-            <input
-              id="email"
-              class="login-form__input"
-              name="email"
-              type="text"
-              placeholder="이메일을 입력해주세요"
-              required
-            />
+            <div class="login-form__input-wrap">
+              <input
+                id="email"
+                class="login-form__input js-input"
+                name="email"
+                type="text"
+                placeholder="이메일을 입력해주세요"
+              />
+              <div class="js-email-error js-error-message"></div>
+            </div>
 
             <label for="password" class="login-form__label">비밀번호</label>
-            <input
-              id="password"
-              class="login-form__input"
-              name="password"
-              type="password"
-              minlength="8"
-              placeholder="비밀번호를 입력해주세요"
-              required
-            />
-            <!-- view, hidden 아이콘 추가 -->
-            <button type="submit" class="login-form__btn">로그인</button>
+            <div class="login-form__input-wrap">
+              <input
+                id="password"
+                class="login-form__input login-form__pw js-input js-pw"
+                name="password"
+                type="password"
+                minlength="8"
+                placeholder="비밀번호를 입력해주세요"
+              />
+
+              <img
+                class="login-form__icon js-pw-icon"
+                src="/images/icon/pw-hidden.png"
+                alt="비밀번호 숨기기 버튼"
+              />
```

Comment:
P2:
상호작용 요소이므로 button 태그를 추천드립니다.

```suggestion
              <button type='button' class='js-pw-icon'>
                <img
                  class="login-form__icon"
                  src="/images/icon/pw-hidden.png"
                  alt="비밀번호 숨기기 버튼"
                />
              </button>
```